support-e2e/tests/utils/users.ts

```diff
@@ -13,4 +13,4 @@ function generateString(length:number) {
 
 export const firstName = () => `${generateString(5)}TestF`;
 export const lastName = () => `${generateString(5)}TestL`;
-export const email = () => `test${uuidv4()}@theguardian.com`;
+export const email = () => `test.e2e.supporter.revenue+${uuidv4()}@theguardian.com`;
```

Comment:
Agree that a clash with a genuine email seems very unlikely!

Reply:
I kept the `test` prefix too as I wondered if there might be some check somewhere that uses it.